### PR TITLE
NMS-14444: Move Restore & Backup topics to /deployment/upgrade

### PR DESCRIPTION
**Pull request by emarsh1995**
- Moved backup and restore docs (previously at /operation/admin/backup and /operation/admin/restore, respectively) to /deployment/upgrade directory. 
- Updated nav files in /operation and /deployment. 
- Reviewed docs in /deployment.

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-14444

### CODE REVIEW

**Review by mmahacek — DISMISSED** [review]
I got nothing. LGTM.

File: docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc [hunk 1]
```diff
@@ -57,17 +61,17 @@ yum-config-manager --disable opennms-repo-stable-*
...
-. Execute `runjava` to update which JVM {page-component-title} will use:
+. Execute `runjava` to set the JVM that {page-component-title} will use:
 +
 [source, console]
 ----
 ${OPENNMS_HOME}/bin/runjava -s
 ----
 
-. Check for configuration file changes, and update accordingly, using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[Idenfity Changed Configuration Files].
+. Check for configuration file changes, and update accordingly using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[idenfity changed configuration files].
```
Comment (Bonrob2) [inline docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc:71]:
```suggestion
. Check for configuration file changes, and update accordingly using the files you backed up in xref:deployment:upgrade/diff.adoc#run_diff[identify changed configuration files].
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
GAH! Found one typo from before. Otherwise, looks good.

File: docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc (hunk 1 above)
Comment (emarsh1995) [reply docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc:71]: Good catch! I don't even know how I missed that. Thank you!

Comment (Bonrob2) [reply docs/modules/deployment/pages/upgrade/rhel/up-rhel.adoc:71]: I don't know how we missed before. There were lots of eyes on that section. :-)

**Review by Bonrob2 — APPROVED** [review]
LGTM! Since Mark also approved, you're good to merge.

**Pull request merged**